scripts/fuse/elektra_fuse/elektra_util.py

```diff
@@ -0,0 +1,196 @@
+from pathlib import Path
+import os, re, errno
+
+import kdb
+
+
+#TODO: Add 'default:' and 'proc:'. These namespaces do not work out of the box like "user:", "sysem:", ... but need extra treatment.
```

Comment:
See #3651

Reply:
Could you further elaborate on how #3651 pertains to this?

Reply:
`proc:` is now available as a read-only namespace.